### DISCUSSION TITLE
最新版本bug

### DISCUSSION THREAD

**Discussion by aaronameixa** [densen2014/docs#23] [Q&A]
无法修改发票号。 我改了发票号码一旦结单自动改成我当时发票号下一位数字。
现在要在哪里设置？

**Comment by aaronameixa** [comment]
版本 0605

**Comment by densen2014** [comment]
最新版已经修复了。 其实改了号码， 只是UI没刷新。

https://file2.app1.es/POS/230611-beta.exe
